hrldas/HRLDAS_forcing/run/examples/GLDAS/extract_gldas.py

```diff
@@ -38,30 +38,30 @@
             modays = leap_days
         if (yrs[yy] == "00" or "04" or "08" or "12" or "16"):
             modays = leap_days
-        for julday in range(day_start+1,day_end+2,1):
+        for julday in range(day_start,day_end+1,1):
             for mo in range(0,12,1):
                 if (julday>modays[mo] and julday<=modays[mo+1]):
-		    day = (julday - modays[mo])
+                    day = (julday - modays[mo])
```

Comment:
This new "green" line seems exactly the same as the original one deleted "red" here. What has changed? Similar for the two places below?

Reply:
Hi Cenlin, the changes are related to consistent use of tabs or spaces in python, i.e. "TabError: inconsistent use of tabs and spaces in indentation using python".

Reply:
Oh, I see. thanks! Is this a change to be compatible with the latest python version (e.g., after v3)? Would this also work with the older python version (e.g., v2.7)?

Reply:
The usage of tabs and spaces for indenting in one file is not allowed in python3, but allowed in python2. That's why some users with python3 had this issue before. Now should be OK.